extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/handler/ZoneTemperatureControlHandler.java

```diff
@@ -28,6 +28,7 @@
 import org.eclipse.smarthome.binding.digitalstrom.internal.lib.climate.jsonResponseContainer.impl.TemperatureControlStatus;
 import org.eclipse.smarthome.binding.digitalstrom.internal.lib.listener.TemperatureControlStatusListener;
 import org.eclipse.smarthome.binding.digitalstrom.internal.lib.manager.StructureManager;
+import org.eclipse.smarthome.binding.digitalstrom.internal.lib.manager.impl.TemperatureControlManager;
```

Comment:
how is this import related to the fixed log line?

Reply:
You are absolutely right, it is not! My "Organize imports"option in Save Actions in Eclipse was selected and with saving changes it must have added these imports.. I did not consider it! Thank you!